.docksal/commands/cypress

```diff
@@ -14,4 +15,8 @@ export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || pr
 
 cd ${PROJECT_ROOT}/tests/cypress
 nvm use
-CYPRESS_BASE_URL="http://${VIRTUAL_HOST}" ./node_modules/.bin/cypress  "$@"
+
+ENVIRONMENT=${2:-"$VIRTUAL_HOST"}
```

Comment:
I think this needs to be

`ENVIRONMENT=${2:-"https://${VIRTUAL_HOST}"}`

Reply:
Updated.